src/components/calcite-checkbox/calcite-checkbox.tsx

```diff
@@ -133,7 +133,11 @@ export class CalciteCheckbox {
   /** Emitted when the checkbox checked status changes */
   @Event() calciteCheckboxChange: EventEmitter;
 
-  /** Emitted when the checkbox focused state changes */
+  /**
+   * Emitted when the checkbox focused state changes
+   *
+   * @internal
+   */
   @Event() calciteCheckboxFocusedChange: EventEmitter;
```

Comment:
Don't see any harm why this can't be a public event. ☝️

Reply:
I'm wondering if we will have more focus events within a component we should come up with one focus event per component like...

```
calciteCheckboxFocus {
  detail: {
    focusId: "checkbox", // node focused?
    target: HTMLInputElement
  }
}
```